backend/database/factories/UserFactory.php

```diff
@@ -23,10 +23,9 @@ class UserFactory extends Factory
     public function definition()
     {
         return [
-            'name' => $this->faker->name,
             'email' => $this->faker->unique()->safeEmail,
             'email_verified_at' => now(),
-            'password' => '$2y$10$92IXUNpkjO0rOQ5byMi.Ye4oKoEa3Ro9llC/.og/at2.uheWG/igi', // password
+            'password' => bcrypt('password'),
```

Comment:
use Hash::make instead of Bcrypt. It's same functionality but the Hash one improves the readability of the code. Also, the Hash is in the Documentation of Laravel.

Reply:
Done sir.